src/modules/rekor/components/rekor_explorer.tsx

```diff
@@ -1,3 +1,4 @@
+import moment from 'moment';
```

Comment:
The formatter will likely complain about the single quotes ('), we have it configured to require double quotes ("). 

You can also run `yarn prettier . --write` to let the formatter fix everything for you :)